### PR TITLE
Update README (GA)

### PR DESCRIPTION
**Pull request by misscoded**
### Type of change (place an x in the [ ] that applies)

- [ ] New sample
- [ ] New feature
- [ ] Bug fix
- [x] Documentation

### Summary

Update README with updated structure.

### Requirements (place an x in each [ ] that applies)

- [x] I’ve checked my submission against the Samples Checklist to ensure it complies with all standards
- [x] I have ensured the changes I am contributing align with existing patterns and have tested and linted my code
- [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,59 +1,60 @@
-# Workflows for GitHub Sample App
+# GitHub Functions
 
-This app brings oft-used GitHub functionality - such as creating new issues - to
-Slack using functions and workflows.
+This sample demonstrates how to create automation in Slack using GitHub.
```
Comment (e-zim) [inline README.md:3]:
```suggestion
This sample demonstrates automations in Slack that interact with the GitHub API.
```

Feel free to adjust this or ignore, but I think guides do more wholistic job of demonstrating "how to create" than the sample. Maybe we also include [the guide](https://api.slack.com/tutorials/tracks/create-github-issues-in-workflows) in the README?

File: README.md [hunk 2]
```diff
@@ -127,127 +128,160 @@ tokens.
...
-the `slack activity` command:
+Activity logs of your application can be viewed live and as they occur with the
+following command:
 
 ```zsh
-$ slack activity
+$ slack activity --tail
 ```
 
 ## Project Structure
```
Comment (e-zim) [inline README.md:237]: Could we also include the `external_auth/` section found on [the timesheet approval sample](https://github.com/slack-samples/deno-timesheet-approval#external_auth)?

File: README.md [hunk 3]
```diff
@@ -127,127 +128,160 @@ tokens.
...
+development and deployed apps.
 
-### `slack.json`
+### `datastores/`
 
-Used by the CLI to interact with the project's SDK dependencies. It contains
-script hooks that are executed by the CLI and implemented by the SDK.
+[Datastores](https://api.slack.com/automation/datastores) securely store data
+for your application on Slack infrastructure. Required scopes to use datastores
+include `datastore:write` and `datastore:read`.
```
Comment (e-zim) [inline README.md:248]: This sample doesn't include this, so I think this section can be removed!

File: README.md [hunk 4]
```diff
@@ -127,127 +128,160 @@ tokens.
...
-Shortcut URL. Copy and paste this URL into a channel as a message, or add it as
-a bookmark in a channel of the workspace you selected.
+## Datastores
 
-**Note: this link won't run the workflow until the app is either running locally
-or deployed!** Read on to learn how to run your app locally and eventually
-deploy it to Slack hosting.
+For storing data related to your app, datastores offer secure storage on Slack
+infrastructure. The use of a datastore requires the
+`datastore:write`/`datastore:read` scopes to be present in your manifest.
```
Comment (e-zim) [inline README.md:201]: I believe this section can be removed for this sample! 🚫🗄️!

File: README.md [hunk 5]
```diff
@@ -1,59 +1,60 @@
...
+- [Included Workflows](#included-workflows)
 - [Setup](#setup)
   - [Install the Slack CLI](#install-the-slack-cli)
   - [Clone the Sample App](#clone-the-sample-app)
   - [Register a GitHub App](#register-a-github-app)
   - [Configure Outgoing Domains](#configure-outgoing-domains)
-- [Create a Link Trigger](#create-a-link-trigger)
-- [Running Your Project Locally](#running-your-project-locally)
+- [Creating Triggers](#creating-triggers)
+- [Datastores](#datastores)
```
Comment (e-zim) [inline README.md:14]:
```suggestion
```

Some comments on this are below!

File: README.md [hunk 6]
```diff
@@ -1,59 +1,60 @@
-# Workflows for GitHub Sample App
```
Comment (e-zim) [inline README.md:1]: A very welcomed change to this header 🙌

**Review by e-zim — CHANGES_REQUESTED** [review]
Thank you once more for making all of these improvements! 🙇

Left a few comments on the datastore sections included for this sample that I think are worth removing, and one on a small note about potentially including the guide from the docs.

Everything else is really solid, so will following any changes with an approval!

**Comment by misscoded** [comment]
No longer relevant. Closing.

**Pull request closed without merging**